tests/jballerina-unit-test/src/test/resources/test-src/taintchecking/propagation/param-status-with-native-invocations-negative.bal

```diff
@@ -14,12 +14,13 @@
 // specific language governing permissions and limitations
 // under the License.
 
-import ballerina/io;
-import ballerina/config;
-
 public function main(string... args) returns error? {
     string key = args[0];
-    io:println(key);
-    boolean b = config:contains(key);
+    boolean b = contains(key);
     return;
 }
+
```

Comment:
Shall we remove the unnecessary line here? Same happens in the other bal file too.